setup.py

```diff
@@ -1372,6 +1372,13 @@ def main():
                 "lib/*.lib",
             ]
         )
+        aotriton_image_path = os.path.join(lib_path, "aotriton.images")
```

Comment:
In 0.8 we moved GPU kernels in to separate files to counter the bloating size of share object file
This is the subdirectory that AOTriton puts all GPU kernels, it must be on the same directory of `libaotriton.so`

Reply:
@malfet to take a look at this change especially. We have additional files in aotriton that would need to be packaged with the pytorch installation